.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,6 +3,7 @@ Thank you for opening a Pull Request! Before submitting your PR, there are a few
 - [ ] Ensure the tests and linter pass
 - [ ] Code coverage does not decrease (if any source code was changed)
 - [ ] Appropriate docs were updated (if necessary)
+- [ ] Rollback plan is reviewed and LGTMed
```

Comment:
Owlbot is going to revert this - you can add it to our excludes list: https://github.com/googleapis/java-bigtable/blob/main/owlbot.py#L97

Reply:
Done! Thanks!